### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by BosEriko**
add issue templates for bug reports and feature requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Please search for existing issues to avoid creating duplicates. -->
+
+<!-- Use Help > Report Issue to prefill these. -->
+- Firebase Swissknife Version:
+- Browser:
+- Browser Version:
```
Comment (tstreamDOTh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Browser & Browser Version is irrelevant, you can ask for`node` version & `firebase-functions` version

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Steps to Reproduce:
+
+1.
+2.
+
+Does this issue occur when all extensions are disabled?: Yes/No
```
Comment (tstreamDOTh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This is irrelevant, you can remove this

**Comment by BosEriko** [comment]
@tstreamDOTh Hello! Sorry for the late reply. I've updated the PR. :)

**Comment by tstreamDOTh** [comment]
@BosEriko Thanks for contributing, please do star ⭐️ the repo if you liked it!

**Comment by tstreamDOTh** [comment]
@BosEriko Thanks for contributing, please do star ⭐️ the repo if you liked it!

**Comment by tstreamDOTh** [comment]
@all-contributors please add @BosEriko  for documentation

**Pull request merged**